core/backend/src/ElementTreeWalker.ts

```diff
@@ -0,0 +1,432 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Elements
+ */
+import { assert, DbResult, Id64Array, Id64String, Logger, LogLevel } from "@itwin/core-bentley";
+import { IModel } from "@itwin/core-common";
+import { BackendLoggerCategory } from "./BackendLoggerCategory";
+import { SubCategory } from "./Category";
+import { DefinitionElement, DefinitionPartition, Element, Subject } from "./Element";
+import { IModelDb } from "./IModelDb";
+import { DefinitionModel, Model } from "./Model";
+
+export const loggerCategory = `${BackendLoggerCategory.IModelDb}.ElementTreeWalker`;
+
+interface ElementTreeWalkerModelInfo { model: Model, isDefinitionModel: boolean }
```

Comment:
Must be exported because it is used by public property of exported type ElementTreeWalkerScope.

Reply:
done